imageresizerapp/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,7 @@
+﻿Install-ChocolateyPackage `
+    -PackageName 'imageresizerapp' `
+    -FileType 'EXE' `
+    -SilentArgs '/quiet /norestart' `
```

Comment:
I don't remember if I added that because this installer suggests or forces a restart. A lot of non-MSI installers seem to require restart suppression. If you know it's not necessary, you can remove it (you can test that).